tapdb/asset_minting.go

```diff
@@ -1174,6 +1174,98 @@ func (a *AssetMintingStore) CommitBatchTx(ctx context.Context,
 	})
 }
 
+// AddSeedlingGroups stores the asset groups for seedlings associated with a
+// batch.
+func (a *AssetMintingStore) AddSeedlingGroups(ctx context.Context,
```

Comment:
Can't this be done when we commit the batch to disk to mark it as finalized?

Reply:
As in defer it to the end of `finalizeBatch` instead of at the end of `sealBatch`? If we don't write it at the end of `sealBatch` then we'd still need to persist the populated asset groups somewhere, like the batch state itself.

Though as-is we'd also want to delete the group if the batch gets cancelled, which isn't implemented.

Right now this data gets written at the end of the caretaker `BatchStateFrozen` in `AddSproutsToBatch`.